examples/deployer/.env.local.example

```diff
@@ -2,5 +2,5 @@ API_PUBLIC_KEY="<Turnkey API Public Key (that starts with 02 or 03)>"
 API_PRIVATE_KEY="<Turnkey API Private Key>"
 BASE_URL="https://api.turnkey.com"
 ORGANIZATION_ID="<Turnkey organization ID>"
-SIGN_WITH="<Turnkey Wallet Account Address, Private Key Address, or Private Key ID>" # if blank, we will create a wallet for you
+PRIVATE_KEY_ID="<Turnkey (crypto) private key ID>" # if you leave it blank, we'll create one for you via calling the Turnkey API
```

Comment:
i think we may still need `SIGN_WITH`, since that's what's used in the primary script https://github.com/tkhq/sdk/blob/9041b6a57f515ea8ba2105150ee5bbe251443c1a/examples/deployer/src/index.ts#L36.

generally, i think we can move completely off of `PRIVATE_KEY_ID` in most of these examples, which means we could delete `/createNewEthereumPrivateKey.ts` from most places as well. don't need to comb through every example now, but perhaps for now we can leave this env var as `SIGN_WITH`.